test/importing.coffee

```diff
@@ -2,20 +2,6 @@
 # ---------
 
 unless window? or testingBrowser?
-  test "coffeescript modules can be imported and executed", ->
```

Comment:
I deleted it for now because it was causing issues with the async test-suite. With the new test-suite there can't be nested or recursive `test` calls.